modules/@apostrophecms/module/lib/events.js

```diff
@@ -82,6 +91,7 @@ module.exports = function(self) {
       if (_.camelCase(eventName) === _.camelCase(handlerName)) {
         throw new Error('The handler name ' + handlerName + ' is essentially identical to the event name, ' + eventName + '. To prevent conflicts with other handlers, you must give it a distinct name expressing what it actually does with the event.');
       }
+      eventName = self.apos.eventAliases[`${moduleName}:${eventName}`] || eventName;
```

Comment:
Evaluated here in `on` so that we don't impose a performance hit in actual handlers.